sqlx-core/Cargo.toml

```diff
@@ -93,11 +93,14 @@ _rt-actix = ["tokio-stream"]
 _rt-async-std = []
 _rt-tokio = ["tokio-stream"]
 _tls-native-tls = []
-_tls-rustls = ["rustls", "webpki", "webpki-roots"]
+_tls-rustls = ["rustls", "rustls-pemfile", "webpki", "webpki-roots"]
 
 # support offline/decoupled building (enables serialization of `Describe`)
 offline = ["serde", "either/serde"]
 
+# openssl
+openssl-native = [ "openssl" ]
```

Comment:
I don't really see the reasoning behind giving this a different feature name. Why not just `openssl` like in the `sqlx` crate?

If you tried to define an `openssl` feature but got an error, it's because you don't need to explicitly define features for `optional = true` dependencies.